devtools/platform-descriptor-json/src/main/resources/codestarts/azure-functions-http-example/base/src/main/resources/application.yml

```diff
@@ -0,0 +1 @@
+quarkus.http.root-path: "/api"
```

Comment:
this will make all other examples fails right?

@maxandersen how should we deal with this?

Reply:
hmm - what is that needed for @patriot1burke ? looks more like a final polish users can decide to do ?

Reply:
If Rest Assured is used, it automatically prepends quarkus.http.root-path.  So the test code doesn't need to change.

Reply:
A "/api" base is required for azure functions though :(

Reply:
@patriot1burke could you make `/api/..` part of the specific path (not defined globally)?

Reply:
@ia3andy I dont want to do it that way.  The azure functions codestart does not generate any java code.  The resteasy, vertx web, funqy, or servlet codestart will generate the java code.  Plus, putting "/api" explicitly in the java code is not how I want the codestart to work.  "/api" is the default root path for azure functions.  I don't want users to have to change their code if the root context changes.

Reply:
hmmm - @patriot1burke I am sorry to say that, but, we aren't ready to have dependent codestarts like this and I am not sure we will be. Example Codestarts are supposed to be 99% independent and standalone to avoid exponential complexity problems. Your example needs to either demo `RESTEasy` of `Vertx-Web` but as a standalone example. We don't have it yet but we could at some point add a way to activate some examples on combination of extensions.

Reply:
Imagine if just the `path` of the `RESTEasy` codestarts change, it would break those tests (which by the way is the case since it's namespaced to `/resteasy/hello`). 

I know this is going to require sacrifices, but having stable/working generated example code is key here IMHO.

Reply:
@ia3andy Your example is wrong.  

If you generate a resteasy-only codestart and a "resteasy+azure-functions-http" codestart, the java code is exactly the same and it works for both.  Again, Rest Assured testing framework automatically prepends the root-path so that you don't have to code it.  So long as the generated test code uses Rest Assured, everything will work fine.

Reply:
@patriot1burke your test is referencing `/hello` this is the Path I am talking about (not the root path)